shaders/src/shading_unlit.fs

```diff
@@ -34,6 +34,8 @@ vec4 evaluateMaterial(const MaterialInputs material) {
     color.a = computeMaskedAlpha(color.a);
     if (color.a <= 0.0) {
         discard;
+    } else {
+        color.a = 1.0;
```

Comment:
This should `if (frameUniforms.transparentTarget) color.a = 1.0`.

Reply:
@prideout you'll need to refactor things a bit to make that work.

Renderer.cpp already has a `needsAlphaChannel` but it's computed way after the call to `view.prepare()` which is where the per-view UBO is set. You'll need to move the calculation of `needsAlphaChannel` up and pass it to `prepare`. I think that should do the trick.